### ISSUE TITLE
Guides do not show up, error when trying to create new guide

### ISSUE THREAD

**Issue by boblmartens** [subjectsplus/SubjectsPlus#1357]
Updated to v4.3 and now all of the guides do not show up. It looks like the queries never complete.

When I create a new guide I get the following error:

`Someone Has Blundered!
We have a problem with the title query: INSERT INTO subject (subject, shortform, description, keywords, redirect_url, active, type, header, extra, course_code, instructor) VALUES ( 'Test', 'TEst', '', '', '', '1', 'Course', 'default', '{\"maincol\":\"\"}', '', '' )`

**Comment by abelfacenda** [comment]
Hello @boblmartens. What was your previous version number?

**Comment by boblmartens** [comment]
I believe that we were on 4.2.1, but perhaps a pulled git version. One thing I have found is that we do not have columns for course_number of instructor in our guides table. Would that cause some oddities?

**Comment by agdarby** [comment]
Yes, I think you just need to add those columns.  I'm out of the
office/country right now, and looking on a phone, but I think this file
should have the information you need buried in it:

https://github.com/subjectsplus/SubjectsPlus/blob/master/lib/SubjectsPlus/Control/Installer.php

You might also want to use the SP4 final branch, since that has some bug
fixes we just completed.  It has only had light QA so far 😮 but seems ok.

**Comment by boblmartens** [comment]
Thanks. We were able to apply the SQL queries to add the needed columns and things came back right away.

Sincerely,
Bob Martens

http://bobmartens.net
@boblmartens

**Comment by boblmartens** [comment]
I'm not sure if it was tied to the fact we had a pre-release version from Git before and then move to the released version so the version-checking did not work for the upgrade.
